pyproject.toml

```diff
@@ -179,4 +179,3 @@ classmethod-decorators = ["classmethod", "pydantic.validator", "pydantic.root_va
 "examples/**" = ["INP001"]
 "olive/engine/packaging/sample_code/ONNXModel/python/**" = ["INP001"]
 "scripts/**" = ["INP001", "PTH123"]
-"examples/directml/llama_v2/chat_app/**"
```

Comment:
the item here is a key=["rules1 to be ignored", "rules2 to be ignored", ...]

Reply:
But this is for ruff right? How can I stop `absolufy-import` from modifying files in that folder since it's a different hook?

Reply:
do you mean ignore for pylint? the ignore-paths is what you are looking for 
https://pylint.pycqa.org/en/latest/user_guide/configuration/all-options.html
 https://github.com/pylint-dev/pylint/blob/3a4bb28304a276b47cf3b6f0ad9d3d5e39225813/examples/pyproject.toml#L45

Reply:
"examples/directml/llama_v2/chat_app/**" = ["TID252"]

https://docs.astral.sh/ruff/rules/#flake8-simplify-sim